src/libs/E2E/apiMocks/openApp.ts

```diff
@@ -1,4 +1,7 @@
-export default () => ({
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
There is a [discussion](https://github.com/Expensify/App/pull/27709#discussion_r1346595804)
which I assume that will resolve issues with bad naming convention and we will be able to sent proper vars names to backed. Just an FYI

Reply:
@kubabutkiewicz In this file bad naming is caused by mocked data fields like `report_98258458`, for example, so I guess we can't do anything